### ISSUE TITLE
.cmx files are not installed

### ISSUE THREAD

**Issue by emillon** [cryptosense/key-parsers#17]
In 4.03 the following warning is emitted when `key-parsers` is linked in:

```
Warning 58: no cmx file was found in path for module Key_parsers, and its interface was not compiled with -opaque
```

You can fix it either by adding `true: opaque` or installing the .cmx files (preferred). `topkg` will do this automatically if you want to give it a try :)

**Comment by NathanReb** [comment]
Let's give `topkg` a try!

**Comment by NathanReb** [comment]
Fixed in [v0.6.1](https://github.com/cryptosense/key-parsers/releases/tag/v0.6.1) !
